functions/packages.bash

```diff
@@ -301,7 +301,7 @@ mqtt_setup() {
     if cond_redirect rm -f /etc/mosquitto/passwd; then echo "OK"; else echo "FAILED"; return 1; fi
   fi
   if ! zram_dependency install mosquitto; then return 1; fi
-  if [[ -s /etc/ztab ]] && ! mkdir -p /opt/zram/log.bind/mosquitto /var/log/mosquitto && chown mosquitto /opt/zram/log.bind/mosquitto /var/log/mosquitto; then echo "FAILED (create zram logdir)"; return 1; fi
+  if zram_is_installed && ! mkdir -p /opt/zram/log.bind/mosquitto /var/log/mosquitto && chown mosquitto /opt/zram/log.bind/mosquitto /var/log/mosquitto; then echo "FAILED (create zram logdir)"; return 1; fi
   echo -n "$(timestamp) [openHABian] Setting up MQTT service... "
```

Comment:
"mosquitto MQTT broker service" or the like

Reply:
I'm confused what your referring to here, do you want me to change the text on line 305? if so that should probably be a separate PR as it is very unrelated.

Reply:
yes line 305
you can make it another PR but in here it's fine, too

Reply:
fixed